ui-ngx/src/app/modules/home/components/dashboard-page/layout/manage-dashboard-layouts-dialog.component.html

```diff
@@ -27,29 +27,125 @@ <h2 translate>layout.manage</h2>
         </mat-checkbox>
       </div>
-      <div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="8px">
-        <button fxFlex
-                type="button" mat-raised-button color="primary"
-                class="tb-layout-button"
-                (click)="openLayoutSettings('main')">
-          <span translate>layout.main</span>
-        </button>
-        <button fxFlex [fxShow]="layoutsFormGroup.get('right').value"
-                type="button" mat-raised-button color="primary"
-                class="tb-layout-button"
-                (click)="openLayoutSettings('right')">
-          <span translate>layout.right</span>
-        </button>
+      <div fxLayout="column" fxLayoutAlign="start center" fxLayoutGap="8px">
+        <div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="8px" style="width: 100%;">
+          <button fxFlex
+                  type="button"
+                  mat-raised-button
+                  color="primary"
+                  class="tb-layout-button"
+                  (click)="openLayoutSettings('main')"
+                  [ngStyle]="buttonStyle('main')">
+            <span >{{ (layoutsFormGroup.value.right ? 'layout.left' :'layout.main')  | translate }}</span>
+          </button>
+          <button fxFlex [fxShow]="layoutsFormGroup.get('right').value"
+                  type="button"
+                  mat-raised-button
+                  color="primary"
+                  class="tb-layout-button"
+                  (click)="openLayoutSettings('right')"
+                  [ngStyle]="buttonStyle('right')">
+            <span >{{ 'layout.right' | translate }}</span>
+          </button>
+        </div>
+        <div fxLayout="column"
+             fxLayoutAlign="start center"
+             fxLayoutGap="8px"
+             [fxShow]="layoutsFormGroup.get('right').value">
+          <mat-button-toggle-group aria-label="Select width value type"
+                                   formControlName="type"
+                                   style="width: 100%;">
+            <mat-button-toggle style="width: 50%;" value="percentage">
+              {{ 'layout.percentage-width' | translate }}
+            </mat-button-toggle>
+            <mat-button-toggle style="width: 50%;" value="fixed">
+              {{ 'layout.fixed-width' | translate }}
+            </mat-button-toggle>
+          </mat-button-toggle-group>
+          <div [fxShow]="layoutsFormGroup.get('type').value === 'percentage'">
+            <div
+                 fxLayoutAlign="start center"
+                 fxLayoutGap="8px"
+                 style="width: 100%;">
+              <mat-slider min="10"
+                          step="1"
+                          style="width: 100%;"
+                          max="90"
+                          formControlName="leftWidthPercentage"
+                          thumbLabel
+                          [value]="layoutsFormGroup.value.leftWidthPercentage"
```

Comment:
It is bad practice to transfer value to the input element when using reactive forms.